src/hooks/useArweaveCompositeProvider/useArweaveCompositeProvider.tsx

```diff
@@ -10,7 +10,7 @@ import eventEmitter from '../../utils/events';
 
 const PDNS_SERVICE_API =
   process.env.VITE_ARNS_SERVICE_API ?? 'https://dev.pdns.app';
-const ARWEAVE_HOST = process.env.VITE_ARWEAVE_HOST ?? 'ar-io.dev';
+const ARWEAVE_HOST = process.env.VITE_ARWEAVE_HOST ?? 'arweave.net';
```

Comment:
is this intentional?

Reply:
oops, no

Reply:
fixed in https://github.com/ar-io/pdns-react/pull/229/commits/770ee8aa94514c233cac10ecee55f3a26abc37a7